cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -89,6 +90,8 @@ func main() {
 	flag.Var(utilflag.NewMapStringBool(&featureGates), "feature-gates", "A set of key=value pairs that describe feature gates for alpha/experimental features. "+
 		"Options are:\n"+strings.Join(utilfeature.DefaultFeatureGate.KnownFeatures(), "\n"))
 
+	flag.StringVar(&provisionercontroller.DefaultFSType, "default-fstype", "ext4", "Specify the filesystem type of the volume. If not specified,  external-provisioner will set default as `ext4`.")
```

Comment:
If we're going to do this I would prefer we follow the style of defining the other flags like `metricsPath`

Reply:
@davidz627 the thing to note here is, `defaultFStype` is already defined here https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/controller/controller.go#L119 unlike other variables in this section like `metricsPath`. So, isnt it good to be here ?

Reply:
that can be changed, it's just a variable

Reply:
@davidz627 this comment is addressed. PTAL 
@jsafrane 
@msau42